assets/js/base/components/price-slider/index.js

```diff
@@ -115,12 +121,14 @@ const PriceSlider = ( {
 			if ( isLoading || ! hasValidConstraints ) {
 				return;
 			}
+			const currentMinRange = minRange.current;
+			const currentMaxRange = maxRange.current;
 			const bounds = event.target.getBoundingClientRect();
 			const x = event.clientX - bounds.left;
-			const minWidth = minRange.current.offsetWidth;
-			const minValue = minRange.current.value;
-			const maxWidth = maxRange.current.offsetWidth;
-			const maxValue = maxRange.current.value;
+			const minWidth = currentMinRange?.offsetWidth;
+			const minValue = currentMinRange?.value;
+			const maxWidth = currentMaxRange?.offsetWidth;
+			const maxValue = currentMaxRange?.value;
 
 			const minX = minWidth * ( minValue / maxConstraint );
 			const maxX = maxWidth * ( maxValue / maxConstraint );
```

Comment:
`minWidth` or `maxWidth` might be undefined here right? So this could result in `NaN`. Have you accounted for that in the various subsequent comparisons derived from these values?